source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,204 @@
+
+![Alt: Inspector screenshot demonstrating that when you inspect a product page description, you find the product__sythesis class in the DOM](./assets/inspector-screenshot.png)
+
+Then you can search for that keyword in your `node_modules/front-commerce/src` folder and you will find the right files.
+
+![Alt: Synthesis screenshot that shows where to find the files given a `class` value](./assets/synthesis-screenshot.png)
+
+### Overriding the files
+
+In **this** example, the files to be overridden are `ProductSynthesisFragment.gql` and `Synthesis.js` both located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`. Copy and paste them into `mymodule/web/theme/modules/ProductView/Synthesis`.
+
+### Updating the query
+
+In `mymodule/web/theme/modules/ProductView/Synthesis/ProductSynthesisFragment.gql`, add the following query line. It means that the application will request the `rate` field as well when sending the query. (Depending on the version of Front-Commerce you are using, the content might differ slightly.)
+
+```diff
+#import "theme/components/organisms/Configurator/ProductConfiguratorFragment.gql"
+#import "theme/modules/AddToCart/ProductStockFragment.gql"
+#import "theme/components/atoms/Typography/Price/ProductPriceFragment.gql"
+#import "theme/modules/ProductView/Details/ProductDetailsFragment.gql"
+
+fragment ProductSynthesisFragment on Product {
+  sku
+  name
+  ...ProductPriceFragment
+  description
++ rate
+  ...ProductConfiguratorFragment
+  ...ProductStockFragment
+  ...ProductDetailsFragment
+  configurations {
+    product {
+      sku
+      ...ProductPriceFragment
+    }
+  }
+}
+```
+
+<blockquote class="info">
+GraphQL queries can be split in Fragments and that each fragment is supposed to live next to a component in order to easily request all the data needed for your component. Thus, since we need to display some new data on the `ProductSynthesis` component, we update the `ProductSynthesisFragment`.
+</blockquote>
+
+## Displaying the result on screen
+
+We're almost there! In the file `Synthesis.js` (which generates the product page), you can add some code to display the `rate`. Knowing React.js or at least HTML syntax is necessary to perform this step without impacting other items on the page.
+
+```diff
+// Many more things here
+const ProductSynthesis = props => {
+  const {
+    product
+    // Many more things here
+  } = props;
+  return (
+    <div className="product__synthesis">
+      {/* Many more things here */}
++     <div>Rate: {product.rate} / 5</div>
+      {/* Many more things here */}
+    </div>
+  );
+};
+
+export default ProductSynthesis;
+```
+
+## Conclusion
+
+Here is a screenshot of the final result on the product page.
+
+![Alt: Product details screenshot with the rate of the product](./assets/product-details-screenshot.png)
+
+And here is a screenshot of the GraphQL query.
```

Comment:
This screenshot does not feel really relevant here.
You could use it at ligne 104 instead of the code examples.